M2/Macaulay2/packages/RationalMaps.m2

```diff
@@ -15,7 +15,7 @@ Version => "0.3", Date => "August 11th, 2019", Authors => {
      {Name => "C.J. Bott",
      Email => "cjamesbott@gmail.com"}
 }, --this file is in the public domain
-Headline => "rational maps")
+Headline => "A package for working with rational maps.", DebuggingMode => true, Reload=>true)
```

Comment:
"a package for" is redundant.  Say instead something like "rational maps".

Reply:
Changed to 
rational maps between varieties